### ISSUE TITLE
editorial policy on subdividing workshop series, and individual lesson "weight"

### ISSUE THREAD

**Issue by ctb** [openjournals/jose#13]
hello,

I wanted to expand [this conversation](https://github.com/openjournals/jose-reviews/issues/130#issuecomment-932632378) into forum that is not tied to just the one submission, because IMO it raises some bigger questions.

## questions of the right subdivision

@labarba writes:
>it kind of looks like salami-slicing an educational initiative into a bunch of thin papers. 

We (the [NIH CFDE training and engagement team](https://training.nih-cfde.org/)) are engaged in a multi-year project to improve data reuse and cloud competency amoung biomedical data scientists through training.

As such, we are developing new workshops regularly. This tranche of lessons ([conda](https://github.com/openjournals/jose-reviews/issues/130) and [AWS](https://github.com/openjournals/jose-reviews/issues/132)) are two of our most popular lessons that have been offered a number of times and seem to have little to no overlap with what is currently publicly available elsewhere.

Since they are standalone and reusable, attract good audiences each time we offer them, and have been independently offered and assessed, we thought we would submit them independently. The reviewers seemed to think the conda lesson was useful, too; one reviewer said,
>I learned a lot from your submission and look forward to being able to use and reference it in my own work. 

and another reviewer said
> was able to spend some time reviewing this morning and found that the materials met most requirements and overall seemed high quality - great work!

This second reviewer also requested more information to be added that put us past the word limits (suggesting that _extending_ the scope of the submission by including a larger module would be overstuffing this submission).

Over in https://github.com/openjournals/jose-reviews/issues/132, @jgorzalski says:
>At JOSE we are looking to publish modules and course lessons with their own objectives and activities that could be taught over three to five weeks of a semester. 

Reading the scope instructions with an eye to this, I see:

>The ideal submission size is a course module, although entire courses are also acceptable.

This could usefully be clarified if what you mean by course module is "three to five weeks of a semester"; that's not at all how I interpreted it.

Back to the main point - it's not easy for me, at least, to figure out what the right subdivision is in the case where we are routinely adding new lessons to a long-running training effort where the overall arc and scope is evolving. I (we) felt that a natural subdivision is "independently offered, assessed, and reusable module", which kinda fits with software development module (independently executable, tested, and reusable :).

## questions of the right weight

@labarba says,

>A 2-hr workshop is on the small side for what we view as a "minimum publishable unit," comparing the scholarly effort to a research paper. We want to be sure that a JOSE paper is respected in the authors' track record and CV, and thus aim for more weighty teaching modules.

I explicitly suggested JOSE for our AWS and conda submissions because we did _not_ see this kind of "impact estimation" criteria in the JOSE submission guidelines; see screenshots below.

<img width="847" alt="Screen Shot 2021-10-02 at 7 20 09 AM" src="https://user-images.githubusercontent.com/51016/135853955-6f955db0-00f0-45ca-b422-1a4dfbd43272.png">
<img width="740" alt="Screen Shot 2021-10-02 at 7 20 33 AM" src="https://user-images.githubusercontent.com/51016/135853975-6ce09607-5c87-4026-b4d7-2f85c35ee388.png">

I absolutely respect the right of the editorial board to lay out and refine the scope of the journal over time, as different edge cases show up. I hope that such considerations can be at least acknowledged as somewhat subjective and documented as such in the submission criteria. Right now such language is entirely absent.

My 2 cents is that there's a pretty strong indication from the reviewers that such a narrowing of the journal scope would reduce the utility of JOSE.

## question of editorial language

I hope the editorial board can visit the question of whether to use the somewhat pejorative term "salami slicing" in editorial communications.

thank you!

**Comment by labarba** [comment]
hi @cbt – shall we close this old issue thread? We did have plenty of conversations in the editorial board, and your submissions did proceed. We've struggled a lot with questions of scope, since the start of this journal, and we haven't arrived at all solutions, but the issue of "size" of the submission has been qualified as follows. 

If a learning module is on the small side, consisting of materials for a short tutorial one would give at a conference, for example, we consider it for publication when it has been offered multiple times and has been evaluated in some way. 

Thus, the assessment of "substantial scholarly contribution" (which is not an assessment of _impact_ at all) considers both the amount of learning material developed, as well as the experience delivering it, and efforts evaluating it.
